source/_integrations/humidifier.markdown

```diff
@@ -0,0 +1,84 @@
+---
+title: "Humidifier"
+description: "Instructions on how to setup humidity control devices within Home Assistant."
+logo: home-assistant.png
+ha_category:
+  - Humidifier
+ha_release: "0.111"
+---
```

Comment:
- `ha_domain` is missing
- `ha_quality_scale` is missing

Reply:
Will `internal` do?

Reply:
Yes, since this is an entity component it is considered internal. Please make sure to set it on the integration manifest as well 😉

Reply:
Done as well, thanks!